python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -27,6 +27,11 @@ keys:
       router_id:
         type: str
         description: In IP address format A.B.C.D.
+      timers:
```

Comment:
I would prefer a proper model (dict) for timers. With each of the timer values being a field.

Reply:
Added